### PR TITLE
FOGL-8280 Add missing control API documentation

### PR DESCRIPTION
**Pull request by MarkRiddoch**

### CODE REVIEW

File: docs/control.rst [hunk 1]
```diff
@@ -325,6 +333,150 @@ The dispatcher can also be instructed to run a local automation script, these ar
...
+Following the above example we can add the name of the entry point and select the type type of control request we wish to make from the drop down menu.
+
++-------------------+
+| | control_api_3 | |
++-------------------+
+
+We then enter destination, in this case service, by selecting it from the drop down. We can also enter the service name.
+
++-------------------+
+| | control_api_4 | |
```
Comment (ashish-jabble) [inline docs/control.rst:435]: same as above

File: docs/control.rst [hunk 2]
```diff
@@ -325,6 +333,150 @@ The dispatcher can also be instructed to run a local automation script, these ar
...
++-------------------+
+
+We can add constant and variable parameters to the entry point via the *Parameters* pane of the add entry page
+
++-------------------+
+| | control_api_5 | |
```
Comment (ashish-jabble) [inline docs/control.rst:441]: same as above

File: docs/control.rst [hunk 3]
```diff
@@ -325,6 +333,150 @@ The dispatcher can also be instructed to run a local automation script, these ar
...
++-------------------+
+
+Clicking on the *+ Add new variable* or *+Add new constant* items will add a pair of entry fields to allow you to enter the name and value for the variable or constant.
+
++-------------------+
+| | control_api_6 | |
```
Comment (ashish-jabble) [inline docs/control.rst:447]: same as above

File: docs/control.rst [hunk 4]
```diff
@@ -325,6 +333,150 @@ The dispatcher can also be instructed to run a local automation script, these ar
...
++-------------------+
+
+You may delete a variable or constant by clicking on the *x* icon next to the entry.
+
+The *Execution Access* pane allows control of who may execute the endpoint. Select the *Anonymous* toggle button will allow any user to execute the API. This is not recommended in a production environment.
+
++-------------------+
+| | control_api_7 | |
```
Comment (ashish-jabble) [inline docs/control.rst:455]: same as above

File: docs/control.rst [hunk 5]
```diff
@@ -325,6 +333,150 @@ The dispatcher can also be instructed to run a local automation script, these ar
...
++-------------------+
+
+The *Allow Users* drop down will provides a means to allow limited users to run the entry point and provides a list of defined users within the system to choose from.
+
+Finally a textual description of the operation may be given in the *Description* field.
+
+Clicking on *Save* will save an enable the new API entry point. The new entry point will be displayed on the resultant screen along with any others that have been defined previously.
+
++-------------------+
+| | control_api_8 | |
```
Comment (ashish-jabble) [inline docs/control.rst:465]: same as above

File: docs/control.rst [hunk 6]
```diff
@@ -325,6 +333,150 @@ The dispatcher can also be instructed to run a local automation script, these ar
...
++-------------------+
+
+Clicking on the three vertical dots will display a menu that allows the details of the entry point to be viewed and updated or to delete the new entry point.
+
++-------------------+
+| | control_api_9 | |
```
Comment (ashish-jabble) [inline docs/control.rst:471]: same as above

File: docs/control.rst [hunk 7]
```diff
@@ -325,6 +333,150 @@ The dispatcher can also be instructed to run a local automation script, these ar
...
++-------------------+
+
+It is also possible to execute the entry point from the GUI by clicking on the name of the entry point. You will be prompted to enter values for any variables that have been defined.
+
++--------------------+
+| | control_api_10 | |
```
Comment (ashish-jabble) [inline docs/control.rst:477]: same as above

**Review by ashish-jabble — CHANGES_REQUESTED** [review]
Images are not loading and other comments.

@MarkRiddoch I have pushed the commit https://github.com/fledge-iot/fledge/compare/FOGL-8280...FOGL-8280-doc-fixes?expand=1, if it looks good then you may cherry-pick this commit into this branch. Alternatively we can create PR on top of your's.

**Comment by MarkRiddoch** [comment]
> Images are not loading and other comments.
> 
> @MarkRiddoch I have pushed the commit https://github.com/fledge-iot/fledge/compare/FOGL-8280...FOGL-8280-doc-fixes?expand=1, if it looks good then you may cherry-pick this commit into this branch. Alternatively we can create PR on top of your's.

Sorry, I worked from the bottom up, it makes it easier to not disturb the line numbers, so didn't see this until I had made the changes.

File: docs/control.rst [hunk 8]
```diff
@@ -323,7 +331,154 @@ The dispatcher can also be instructed to run a local automation script, these ar
...
+        "destination"    : "service",
+        "service"        : "camera1",
+        "constants"      : {
+                                "units"    : "cm"
+                           },
+        "variables"      : {
+                                "distance" : "100"
+                           },
+        "allow"          : [ "john", "fred" ],
+        "anonymous"      : "false"
```
Comment (ashish-jabble) [inline docs/control.rst:401]: bool - false

**Review by ashish-jabble — APPROVED** [review]

**Pull request merged**